copy-plugin.js

```diff
@@ -34,7 +34,7 @@ const pluginManifests = {
       pluginId: 'io.safello',
       pluginURL: 'https://safello.com/edge/',
       name: 'Safello',
-      subtitle: 'BTC, ETH, XRP, BCH\nSettlement - Instant \n3 seconds after confirmed payment',
+      subtitle: 'Fiat currencies EUR, SEK, DKK, NOK, GBP, CHF, HUF, PLN\nFee: 4.75%\nSettlement - Instant',
```

Comment:
Only put in the crypto currencies. No need for the Fiat currencies.

Reply:
Fixed